lib/filestorage.js

```diff
@@ -97,43 +97,44 @@ class FileStorage {
   // Delete file from storage
   //   id - <Object>, common.Uint64, id of file
   //   cb - <Function>, callback
-  //     err - <error>
+  //     err - <Error>
   rm(id, cb) {
     const file = this._getFilepath(common.idToPath(id));
     fs.unlink(file, cb);
   }
 
-  _compress(id, compression, cb) {
+  _compress(id, compression, minSize, cb) {
```

Comment:
I don't think `minSize` should be an argument to this function, it will just be an option to Filestorage i.e. `minCompressSize`.